palm/palm_config.py

```diff
@@ -75,10 +84,25 @@ def image_name(self) -> str:
         """
         return self.config.get('image_name') or self.project_root_snake_case
 
-    @property
```

Comment:
is this breaking then, since we no longer have the `self.plugins` attribute? or is that captured elsewhere?

Reply:
Nope, `self.plugins` is defined as an empty list on `__init__`